book/src/volume-cloning.md

```diff
@@ -0,0 +1,45 @@
+# Volume Cloning
+
+## Status
+
+* Kubernetes 1.15: Alpha
+
+## Overview
+
```

Comment:
Maybe start off with one sentence about what cloning is, and then a second sentence pointing to the Kubernetes user docs.

Reply:
done